llvm/lib/Target/X86/X86SelectionDAGInfo.cpp

```diff
@@ -66,8 +66,10 @@ SDValue X86SelectionDAGInfo::EmitTargetCodeForMemset(
   // If not DWORD aligned or size is more than the threshold, call the library.
   // The libc version is likely to be faster for these cases. It can use the
   // address value and run time information about the CPU.
-  if (Alignment < Align(4) || !ConstantSize ||
-      ConstantSize->getZExtValue() > Subtarget.getMaxInlineSizeThreshold()) 
+  if (!ConstantSize ||
```

Comment:
This change does not match the description of the patch.

Reply:
I'm gonna then branch that to another PR because it's needed

https://github.com/llvm/llvm-project/pull/87825